LICENSE.txt

```diff
@@ -30,7 +30,6 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 ====
 
-All files located in the node_modules and external directories are
-externally maintained libraries used by this software which have their
-own licenses; we recommend you read them, as their terms may differ from
-the terms above.
+All files located in the node_modules are externally maintained
+libraries used by this software which have their own licenses; we
+recommend you read them, as their terms may differ from the terms above.
```

Comment:
Missing "directory", should be "All files located in the node_modules directory are [...]"